src/game/delete-round.ts

```diff
@@ -23,7 +49,8 @@ export class DeleteRound extends Round {
         const line = game.gameBuffer.midPointRandomPoint(high);
 
         const lines = new Array(game.state.lineLength).fill("");
-        lines[line] = "                              DELETE ME";
+
+        lines[line] = this.getGameDeleteMeColumOffset(game) + "DELETE ME";
```

Comment:
`#getGameDeleteMeColumOffset` returns a promise (since it is an async function), so you have to await any calls to it:

```js
lines[line] = await this.getGameDeleteMeColumOffset(game) + "DELETE ME";
```